.circleci/config.yml

```diff
@@ -824,11 +824,11 @@ workflows:
       - sync-licenses:
           name: webflow-licenses-sync
           requires:
-            - production-deploy
+            - webflow-fundings-sync
       - sync-industries:
           name: webflow-industries-sync
           requires:
-            - production-deploy
+            - webflow-licenses-sync
```

Comment:
Chained webflow sync jobs together in an attempt to temporarily mitigate the rate limit issue. Rate limit will be addressed in a following ticket.